rules/DowngradePhp81/snippet/array_is_list_closure.php.inc

```diff
@@ -1,18 +0,0 @@
-<?php
```

Comment:
it used at https://github.com/rectorphp/rector-downgrade-php/blob/8df23f0cabdf045dc4e131f8aa9a4f8e3c61295c/rules/DowngradePhp81/Rector/FuncCall/DowngradeArrayIsListRector.php#L125